app/konnect/runtime-manager/runtime-instances/upgrade.md

```diff
@@ -17,9 +17,6 @@ proxy downtime.
 2. Open {% konnect_icon runtimes %} Runtime Manager, then choose a runtime group.
 Make sure that your new runtime instance appears in the list of runtime
 instances, is in _Connected_ status, and that it was last seen _Just Now_.
-For example:
-
-    ![Connected runtimes in Runtime Manager](/assets/images/docs/konnect/konnect-runtimes-connected.png)
```

Comment:
Should the image also be removed from the repo?

Reply:
It's still in use in an old topic under `/konnect/legacy/`, so not yet.